src/content/guides/efk-stack-app-oidc/index.md

```diff
@@ -0,0 +1,365 @@
+
+## Overview
+
+- Register app on azure portal and configure jwt token
+- Configure OIDC backend for Kibana
+- Configure opendistro-security plugin
+- Configure internal-users database
+- Configure roles-mapping to control users permission
+
+## Register app on azure portal and configure jwt token
+### Limitation
+As of writing opendistro security does not support mapping nested values in the jwt-token to roles. Currently it's only possible to specify the key, which contains the values that identify the specific users role identifier.
+
+### Register app on azure portal
+This step can differ depending on the backend provider. Generally there should be a mechanism to define an object that represents the app with attributes to control who has which permissions on it.
+In Azure, this is called an "App registration"
+
+To setup a new "App registration" refer to the [official guide](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app).
+
+For the **redirect URI** the pattern is as follows:
+https://<your.kibana.url>/auth/openid/login
+*This can be changed later on*
+
+#### Configure jwt token
+The goal is to map a users group ID in Azure to a role that defines the users permissions in Kibana. For example, if the user is member of the Azure AD group "kibana-admins", the user obtains the role "admin" in Kibana.
+For this to work, the jwt token issued by Azure need to contain the group IDs of the user. This is done by adding an optional claim to the app-registrations token configuration.
+
+1. In Azure platform navigate to the app-registration create in the previous section
+2. In the sidebar click on "Token configuration"
+3. Click on "+Add groups claim"
+4. Select the group types to include in the token (usually Security groups)
+5. In the "Customize token properties by type" select "Group ID" for every type
+6. Click "Save"
+
+For further reference on how to define groups optional claims, please refer to [microsofts official documentation](https://docs.microsoft.com/en-us/azure/active-directory/develop/active-directory-optional-claims#configuring-groups-optional-claims) on the topic.
+
+How to define and manage the required groups, as well as how to obtain the group IDs of those groups is out of scope for this guide. Check [microsofts documentation on Azure AD](https://docs.microsoft.com/en-us/azure/devops/server/admin/setup-ad-groups?toc=%2Fazure%2Fdevops%2Forganizations%2Ftoc.json&bc=%2Fazure%2Fdevops%2Forganizations%2Fbreadcrumb%2Ftoc.json&view=azure-devops-2020&viewFallbackFrom=azure-devops)
+
+### Configure opendistro-security plugin
+Opendistro-security can be configured through secrets. These secrets need to be present in the apps namespace for the above EFK configuration to successfully deploy. In this section we will setup the target namespace and deploy the `configSecret` named `opendistro-security-config`.
```

Comment:
"for the above EFK configuration to successfully deploy."

What is this referring to? I don't see any EFK configuration above. Should it be "for the below EFK configuration"?